cmd/root.go

```diff
@@ -47,5 +48,12 @@ func init() {
 	rootCmd.AddCommand(factory.Build(cli.WhoamiCommand))
 
 	// pass a list of functions to be run before each command's Execute function
-	cobra.OnInitialize(func() { cli.HandleErr(profile.Load()) })
+	cobra.OnInitialize(func() { handleErr(profile.Load()) })
+}
+
+func handleErr(err error) {
```

Comment:
@tsedgwick ... here's the change that works on this `log.Fatal` handling for the CLI (which you brought up [here](https://github.com/10gen/realm-cli/pull/28#discussion_r529654776)).

figured if you still want to talk about it and/or suggest some changes, here would be the place

Reply:
follow-up: removed this entirely and am just handling these errors in place (the CLI will/should catch-and-log any meaningful errors before they ever reach this point)